CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This version adds a dependency on Swift.
 ### Features
 
 - Properly demangle Swift class name (#2162)
+- SwiftUI performance tracking (#2271)
```

Comment:
```suggestion
- SwiftUI auto performance tracking (#2271)
```

Reply:
Would this be considered "auto performance"?